app/status/views.py

```diff
@@ -1,9 +1,46 @@
-from flask import jsonify
+from flask import jsonify, current_app
 
 from . import status
 from . import utils
+from ..main.helpers.service import ServiceLoader
+from ..main import main
 
 
 @status.route('/_status')
 def status():
-    return jsonify(status="ok", version=utils.get_version_label())
+
+    # ServiceLoader is the only thing that actually connects to the API
+    service_loader = ServiceLoader(
+        main.config['API_URL'],
+        main.config['API_AUTH_TOKEN']
+    )
+
+    api_response = utils.return_response_from_api_status_call(
+        service_loader.status
+    )
+
+    apis_wot_got_errors = []
+
+    if api_response is None or api_response.status_code is not 200:
```

Comment:
Should use != for integer comparison - 'is' behaves strangely for int in Python (it will work in this case for a 200, but not for, say a 404). We should consistently use == and != for int comparison in all our code.
See here for more details: http://stackoverflow.com/questions/306313/pythons-is-operator-behaves-unexpectedly-with-integers